pkg/infrared/server.go

```diff
@@ -213,42 +213,38 @@ type statusResponseProvider struct {
 	statusResponseCache map[uint64]*statusCacheEntry
 }
 
-func (s *statusResponseProvider) requestNewStatusResponseJSON(readPks [2]protocol.Packet) (uint64, status.ResponseJSON, protocol.Packet, error) {
+func (s *statusResponseProvider) requestNewStatusResponseJSON(readPks [2]protocol.Packet) (status.ResponseJSON, protocol.Packet, error) {
 	rc, err := s.server.Dial()
 	if err != nil {
-		return 0, status.ResponseJSON{}, protocol.Packet{}, err
+		return status.ResponseJSON{}, protocol.Packet{}, err
 	}
 
 	if err := rc.WritePackets(readPks[0], readPks[1]); err != nil {
-		return 0, status.ResponseJSON{}, protocol.Packet{}, err
+		return status.ResponseJSON{}, protocol.Packet{}, err
 	}
 
 	var pk protocol.Packet
 	if err := rc.ReadPacket(&pk); err != nil {
-		return 0, status.ResponseJSON{}, protocol.Packet{}, err
+		return status.ResponseJSON{}, protocol.Packet{}, err
 	}
 	rc.Close()
 
-	hash := xxhash.New()
-	pk.WriteTo(hash)
```

Comment:
Is there a reason why we're writing the hash to the package? :thinking:

Reply:
No, I don't think so